Packs/RunZero/Integrations/RunZero/RunZero.py

```diff
@@ -0,0 +1,525 @@
+    message['Asset_Status'] = raw.get('alive', '')
+    message['Address'] = raw.get('service_address', '')
+    message['Transport'] = raw.get('service_transport', '')
+    message['Port'] = raw.get('service_port', 0)
+    message['Protocol'] = raw.get('service_protocol', [])
+    message['VHost'] = raw.get('service_vhost', '')
+    message['Summary'] = raw.get('service_summary', '')
+    message['Hostname'] = raw.get('names', [])
+    message['OS'] = f"{raw.get('os', '')} {raw.get('os_version', '')}"
+    message['Type'] = raw.get('type', '')
+    message['Hardware'] = raw.get('hw', '')
+    message['Outlier'] = raw.get('outlier_score', 0)
+    message['MAC_Vendor'] = raw.get('mac_vendors', [])
+    message['MAC_Age'] = raw.get('newest_mac_age', '')
+    message['MAC'] = raw.get('macs', [])
+    message['OS_EOL'] = raw.get('eol_os', 0)
+    message['Comments'] = raw.get('comments', '')
+    message['Tags'] = raw.get('tags', {})
+    message['Svcs'] = raw.get('service_count', 0)
+    message['TCP'] = raw.get('service_count_tcp', 0)
+    message['UDP'] = raw.get('service_count_udp', 0)
+    message['ICMP'] = raw.get('service_count_icmp', 0)
+    message['ARP'] = raw.get('service_count_arp', 0)
+    message['SW'] = raw.get('software_count', 0)
+    message['Vulns'] = raw.get('vulnerability_count', 0)
+    message['RTT/ms'] = normalize_rtt(raw.get('lowest_rtt', 0))
+    message['Hops'] = raw.get('lowest_ttl', 0)
+    message['Detected'] = raw.get('detected_by', 0)
+    message['First_Seen'] = timestamp_to_datestring(raw.get('first_seen', '') * 1000)
+    message['Last_Seen'] = timestamp_to_datestring(raw.get('last_seen', '') * 1000)
+    message['Explorer'] = raw.get('agent_name', '')
+    message['Hosted_Zone'] = raw.get('hosted_zone_name', '')
+    message['Site'] = raw.get('site_name', '')
+    return [message]
+
+
+def parse_raw_quota_get(raw: dict) -> dict:
+    message = {}
+    message['usage_limit'] = raw.get('created_at', '')
+    message['usage_today'] = raw.get('usage_today', '')
+    message['counter'] = raw.get('counter', '')
+    return message
+
+
+def parse_raw_wireless(raw: dict) -> list:
+    message = {}
+    message['ID'] = raw.get('id', '')
+    message['ESSID'] = raw.get('essid', '')
+    message['BSSID'] = raw.get('bssid', '')
+    message['Vendor'] = raw.get('vendor', '')
+    message['Family'] = raw.get('family', '')
+    message['Type'] = raw.get('type', '')
+    message['Auth'] = raw.get('authentication', '')
+    message['Enc'] = raw.get('encryption', '')
+    message['Sig'] = raw.get('signal', 0)
+    message['Int'] = raw.get('interface', '')
+    message['Additional'] = raw.get('data', {})
+    message['First_seen'] = timestamp_to_datestring(raw.get('created_at', ''))
+    message['Last_seen'] = timestamp_to_datestring(raw.get('last_seen', ''))
+    message['Site'] = raw.get('site_name', '')
+    return [message]
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def asset_search(client: Client, args: dict) -> CommandResults:
```

Comment:
Please add the `_command` suffix to this function name, and to the rest of the command functions.